dnsimple/vanity_name_server.go

```diff
@@ -0,0 +1,65 @@
+package dnsimple
+
+import (
+	"fmt"
+)
+
+// VanityNameServersService handles communication with Vanity Name Servers
+// methods of the DNSimple API.
+//
+// See https://developer.dnsimple.com/v2/vanity/
+type VanityNameServersService struct {
+	client *Client
+}
+
+// VanityNameServer represents data for a single vanity name server
+type VanityNameServer struct {
+	ID        int    `json:"id,omitempty"`
+	Name      string `json:"name,omitempty"`
+	IPv4      string `json:"ipv4,omitempty"`
+	IPv6      string `json:"ipv6,omitempty"`
+	CreatedAt string `json:"created_at,omitempty"`
+	UpdatedAt string `json:"updated_at,omitempty"`
+}
+
+// VanityNameServerResponse represents a response for vanity name server enable and disable operations.
+type VanityNameServerResponse struct {
+	Response
+	Data []VanityNameServer `json:"data"`
+}
+
+func vanityNameServerPath(accountID string, domainID string) string {
+	return fmt.Sprintf("/%v/vanity/%v", accountID, domainID)
+}
+
+// Enable Vanity Name Servers for the given domain
+//
+// See https://developer.dnsimple.com/v2/vanity/#enable
+func (s *VanityNameServersService) Enable(accountID string, domainID string) (*VanityNameServerResponse, error) {
```

Comment:
It should be EnableVanityNameServers. Same for Disable. We have been using qualified names to avoid conflicts.